### PR TITLE
New package: arc-themes-20151214

### PR DESCRIPTION
**Pull request by pbui**

### CODE REVIEW

File: srcpkgs/arc-themes/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+pkgname="arc-themes"
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: First line should be a comment `# Template file for 'arc-themes'`. pkgname should not have quotes around it. Neither should version.

File: srcpkgs/arc-themes/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+wrksrc="arc-theme-${version}"
+short_desc="A flat theme with transparent elements for GTK 2 and 3"
+build_style=fetch
+maintainer="Peter Bui <pnutzh4x0r@gmail.com>"
+license="GPL-3"
+noarch=yes
+build_style="gnu-configure"
+hostmakedepends="pkg-config autoconf automake"
+makedepends="gtk+-devel gtk+3-devel"
+depends="gtk+ gtk2-engines gtk-engine-murrine gnome-themes-standard"
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: Why depend on a specific gtk? The theme works for either.

**Comment by pbui** [comment]
@Vaelatern I fixed the quotes and added the comment.  I'm not sure what you mean by depending on a specific version of gtk+, but I removed gtk2-engines.  From the upstream website, the theme requires gtk+, gtk-engine-murrine, and gnome-themes-standard.  To build and auto-detect the correct gtk+ versions, I believe `makedepends` requires both `gtk+-devel` and `gtk+3-devel`, but I can change that if necessary.

**Comment by Vaelatern** [comment]
That part is fine, I just don't think you need to add them as runtime dependencies.

File: srcpkgs/arc-themes/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'arc-themes'
+pkgname=arc-themes
+version=20151214
+revision=1
+noarch=yes
+wrksrc="arc-theme-${version}"
+short_desc="A flat theme with transparent elements for GTK 2 and 3"
+build_style=fetch
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: Remove this line.

File: srcpkgs/arc-themes/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'arc-themes'
+pkgname=arc-themes
+version=20151214
+revision=1
+noarch=yes
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: Remove this line.

File: srcpkgs/arc-themes/template [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+wrksrc="arc-theme-${version}"
+short_desc="A flat theme with transparent elements for GTK 2 and 3"
+build_style=fetch
+maintainer="Peter Bui <pnutzh4x0r@gmail.com>"
+license="GPL-3"
+noarch=yes
+build_style="gnu-configure"
+hostmakedepends="pkg-config autoconf automake"
+makedepends="gtk+-devel gtk+3-devel"
+depends="gtk+ gtk-engine-murrine gnome-themes-standard"
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: `depends="gtk-engine-murrine gnome-themes-standard"`

File: srcpkgs/arc-themes/template [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+revision=1
+noarch=yes
+wrksrc="arc-theme-${version}"
+short_desc="A flat theme with transparent elements for GTK 2 and 3"
+build_style=fetch
+maintainer="Peter Bui <pnutzh4x0r@gmail.com>"
+license="GPL-3"
+noarch=yes
+build_style="gnu-configure"
+hostmakedepends="pkg-config autoconf automake"
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: `hostmakedepends="pkg-config automake"`

File: srcpkgs/arc-themes/template [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'arc-themes'
+pkgname=arc-themes
+version=20151214
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: Latest release is `20160331` and needs a new checksum for it. Fixes for GTK 3.20.

Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: To be fair, the release updated after you opened this PR.

**Comment by pbui** [comment]
I've made the changes you've requested.  Sorry about the sloppiness.

File: srcpkgs/arc-themes/template [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'arc-themes'
+pkgname=arc-themes
+version=20160331
+revision=1
+noarch=yes
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: Remove this line

Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: Disregard.

**Comment by Vaelatern** [comment]
#4002 was closed, so this shall be as well.

**Comment by xtraeme** [comment]
I refuse to push any THEME available on inet just because it can be packaged.

This theme can be downloaded by the user and unpacked into its home directory.

There's no need to make an xbps package.

Thanks!

**Pull request closed without merging**